src/Runner.Worker/ExecutionContext.cs

```diff
@@ -92,7 +98,7 @@ public interface IExecutionContext : IRunnerService
         void SetGitHubContext(string name, string value);
         void SetOutput(string name, string value, out string reference);
         void SetTimeout(TimeSpan? timeout);
-        void AddIssue(Issue issue, string message = null);
+        void AddIssue(Issue issue, ExecutionContextLogOptions logOptions);
```

Comment:
give callers a mechanism to opt-out of logging entirely for a particular call to `AddIssue`